### ISSUE TITLE
ASM Module doesn't Apply Policy after second run of code

### ISSUE THREAD

**Issue by VDI-Tech-Guy** [F5Networks/f5-ansible#1654]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 bigip_asm_policy_manage

##### ANSIBLE VERSION

```
ansible 2.9.3
  config file = /home/student1/.ansible.cfg
  configured module search path = [u'/home/student1/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Oct 30 2018, 23:45:53) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]
```

##### PYTHON VERSION

```
Python 2.7.5
```

##### BIGIP VERSION

```
Sys::Version
Main Package
  Product     BIG-IP
  Version     13.1.0.7
  Build       0.0.1
  Edition     Point Release 7
  Date        Tue May 15 15:26:30 PDT 2018
```

##### SUMMARY
When running the code for the ASM Role to block IP and/or block urls, i launch the code the first time to provision out the environment and build the ASM policy, and it applies correctly.  when i want to re-run the code with just the policy tag with new blocked urls or ips the policy refuses to apply, and provides the "Apply Policy" link in the BIG-IP.  I have tried many ways to trigger the Apply Policy flag but it doesnt seem to trigger after the first run.

##### STEPS TO REPRODUCE
 Security --> Application Security --> Urls --> disallowed URLS or 
in Big IP:8443 --> Security --> Application Security --> IP Addresses 

then look to far right upper side to see "Apply Policy" issue (attached photo)
-->

```
[student1@ansible ~]$ git clone https://github.com/VDI-Tech-Guy/Roles.git
[student1@ansible Usecase-04-WAF]$ ansible-playbook asm.yaml
[student1@ansible Usecase-04-WAF]$ ansible-playbook asm.yaml --tags "policy"


```

##### EXPECTED RESULTS

Expected Policy to apply with bigip_asm_policy_manage module as per my code

```
#Apply Draft ASM policy
  - name: Activate ASM Policy
    bigip_asm_policy_manage:
      provider: "{{provider}}"
      name: "{{ ASM_Policy_Name }}"
      active: yes
```

There are no issues with execution of the code its how the module reacts.

##### ACTUAL RESULTS

Policy doesnt get applied on 2nd run and have to manually trigger the apply policy via the GUI and couldnt find way to trigger with automation in ansible modules.

There are no issues with execution of the code its how the module reacts.

##### Other Notes
Feel free to reach out to me for environment and testing.  

![2020-02-19_12-52-31](https://user-images.githubusercontent.com/1888195/74870918-cc96a780-5317-11ea-8c23-52cddd9f7697.png)

**Comment by focrensh** [comment]
Tracking with FMFA-439

**Comment by wojtek0806** [comment]
It seems the issue has not been closed, so the short version of the problem is that the policy in question is still active and being reported as such via the REST api, and what the GUI does under the hood is that it first deactivates the policy then reactivates it apply the changes. To do this in ansible you need to run the playbook twice, first to set the policy to `active: no` then set the policy to `active: yes` this will resolve your issue. This is working as intended.

**Comment by focrensh** [comment]
Resolved with 5d1d1d3feab27effc77a65ff0292189198f60a71 (should be 1.4 release)
